src/environments/.env.ts

```diff
@@ -1,3 +1,4 @@
 export default {
-  'npm_package_version': '0.0.0'
+  'mifos_x_version': '0.0.0',
+  'allow_switching_backend_instance': true
```

Comment:
Privilege to switch servers can be withdrawn by setting `allow_switching_backend_instance` to false before build time and specifying the server as `baseApiUrl`.